draft-kazuho-httpbis-priority.md

```diff
@@ -197,24 +199,38 @@ because the server-provided value overrides that provided by the client.  The
 urgency is deemed as `non-blocking`, because the server did not specify the
 parameter.
 
-# Coexistence with HTTP/2 Priorities
+# Coexistence with HTTP/2 Priorities {#coexistence}
 
-When connecting to an HTTP/2 ({{!RFC7540}}) server, a client that uses this
-header-based prioritization scheme SHOULD send a
-`SETTINGS_HEADER_BASED_PRIORITY` settings parameter (0xTBD) with a value of
-one.
+Standard HTTP/2 ({{!RFC7540}}) endpoints use frame-based prioritization, whereby
+a client sends priority information in dedicated fields present in HEADERS and
+PRIORITY frames. A client may instead choose to use header-based prioritization
```

Comment:
may -> might or can?

Reply:
Gah I keep doing this, want the lint tool to check it!

Will fix up soon.